.github/workflows/msbuild.yml

```diff
@@ -23,23 +32,32 @@ jobs:
       with:
         arch: ${{ matrix.arch }}
 
-    - name: Restore from cache and install vcpkg
-      uses: lukka/run-vcpkg@v6
+    - name: Clone openvpn repo
+      uses: actions/checkout@v2
       with:
-        setupOnly: true
-        vcpkgGitCommitId: 'a267ab118c09f56f3dae96c9a4b3410820ad2f0b'
-        additionalCachedPaths: ${{ env.buildDir }}/vcpkg_installed
-        appendedCacheKey: ${{ matrix.arch }}-${{ hashFiles( '**/vcpkg.json' ) }}
+        # TODO: replace with openvpn/openvpn
+        repository: lstipakov/openvpn
```

Comment:
I guess this is because openssl3 port is not merged to openvpn/openvpn yet.

Reply:
Yep, https://patchwork.openvpn.net/patch/2254/

Someone needs to ack it (wink wink).

Reply:
Acked, merged. Repo name fixed.